pyproject.toml

```diff
@@ -172,6 +172,20 @@ Forum = "https://mne.discourse.group/"
 "Source Code" = "https://github.com/mne-tools/mne-python/"
 
 [tool.hatch.build]
+artifacts = [
+    "/mne/data/coil_def.dat",
+    "/mne/data/coil_def_Elekta.dat",
+    "/mne/data/eegbci_checksums.txt",
+    "/mne/data/extinction_coef.mat",
+    "/mne/data/FreeSurferColorLUT.txt",
```

Comment:
Iirc this one and a few others listed here are not necessary, as we only need to list those files that are excluded via gitignore

Reply:
@larsoner i can take over if you want

Reply:
`*.txt` is in `exclude` below so I suspect it would be excluded if we don't have it here

Reply:
I just pushed something but if you want to continue feel free